app/src/main/res/values/strings.xml

```diff
@@ -115,8 +115,9 @@
      Mock: https://user-images.githubusercontent.com/1721875/47298694-b809a880-d618-11e8-9e18-1d6d47d2045d.png -->
     <string name="preference_mozilla_telemetry2">Send usage data</string>
     <!-- Description text that explains the purpose of our telemetry
-      Mock: https://user-images.githubusercontent.com/1721875/47298694-b809a880-d618-11e8-9e18-1d6d47d2045d.png -->
-    <string name="settings_telemetry_description">Mozilla strives to collect only what we need to provide and improve Firefox for Fire TV for everyone</string>
+      Mock: https://user-images.githubusercontent.com/1721875/47298694-b809a880-d618-11e8-9e18-1d6d47d2045d.png
+      %1$s will be replaced with the name of the app (e.g. Firefox for Fire TV) -->
+    <string name="settings_telemetry_description">Mozilla strives to collect only what we need to provide and improve %1$s for everyone</string>
```

Comment:
One note here that you may not be aware of: if we need a string to be re-translated (which is happening in this case, because the string has changed), then you need to change the string name - usually we suffix it with a number (like 1). That way, it'll show up in the Pontoon dashboard.